src/EdFi.AnalyticsMiddleTier.DataStandard33/EPP/PostgreSQL/0000-View-CandidateDim-Create.sql

```diff
@@ -8,48 +8,84 @@ CREATE VIEW analytics.EPP_CandidateDim AS
 	SELECT Candidate.CandidateIdentifier AS CandidateKey
 		,Candidate.FirstName
 		,Candidate.LastSurname
-		,Candidate.SexDescriptorId
-		,COALESCE(CandidateRace.RaceDescriptorId, 0) AS RaceDescriptorId
+		,CAST(Candidate.SexDescriptorId AS VARCHAR) AS SexDescriptorKey
+		,SexDescriptor.CodeValue AS SexDescriptor
+		,COALESCE(CAST(CandidateRace.RaceDescriptorId AS VARCHAR), '') AS RaceDescriptorKey
+		,COALESCE(RaceDescriptor.CodeValue, '') AS RaceDescriptor
 		,COALESCE(Candidate.HispanicLatinoEthnicity, false) AS HispanicLatinoEthnicity
 		,COALESCE(Candidate.EconomicDisadvantaged, false) AS EconomicDisadvantaged
-		,COALESCE(ccy.SchoolYear, 0) AS Cohort
-		,CAST(CASE WHEN ReasonExitedDescriptor.CodeValue = 'Completed' THEN 1 ELSE 0 END AS BIT) ProgramComplete
-		,COALESCE(Student.StudentUSI, 0) AS StudentUSI
+		,COALESCE(CAST(CandidateEducatorPreparationProgramAssociationCohortYear.SchoolYear AS VARCHAR), '') AS Cohort
+		,CAST(CASE 
+			WHEN ReasonExitedDescriptor.CodeValue = 'Completed' 
+				THEN 1 
```

Comment:
Postgres should return boolean (true/false) instead of bit (1/0)

Reply:
Fixed.